app/models/station.rb

```diff
@@ -41,4 +41,42 @@ def self.minimum_dock_count
     minimum(:dock_count)
   end
 
+  def started_rides_count
+    trips_started.count
+  end
+
+  def ended_rides_count
+    trips_ended.count
+  end
+
+  def frequent_destination
+    station_id = trips_started.group("end_station_id").order("count_id DESC").count(:id).first[0]
```

Comment:
See the `count(:id).first[0]` pattern a number of places. Generally looks good, but I might also throw a `limit(1)` on to it: will reduce the amount of data that is passed from our SQL query to Ruby.

Also, double check, but believe `count(:id)` will result in a hash. If that's the case, then `keys.first` might better describe what is happening in this code.